templates/io-package.json.ts

```diff
@@ -61,9 +61,9 @@ export = (async answers => {
 		"keywords": ${JSON.stringify(answers.keywords || getDefaultAnswer("keywords"))},
 		"license": "${licenses[answers.license!].id}",
 		"platform": "Javascript/Node.js",
-		"icon": "${answers.adapterName}.png",
+		"icon": "${answers.adapterName}.${answers.icon?.extension || "png"}",
```

Comment:
Problem is: I can only provide an entire "icon" as the default answer, but we only want the default extension. It seems wrong to me to use the method `getDefaultAnswer()` to return something else but an actual answer (the extension is not the answer, the icon is). And we can't set an icon as we want to use the file in case the icon wasn't defined.

Reply:
We could of course add a `export const defaultIconExtension = "png"` somewhere.